<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>等线 Light</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="maintable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>主界面</string>
     </attribute>
     <widget class="QLineEdit" name="GetNameText">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>651</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>PlantsVsZombies.exe</string>
      </property>
     </widget>
     <widget class="QLabel" name="miantitel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>781</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>欢迎使用植物大战僵尸：植物&amp;僵尸自动布阵器</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="tips_one">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>571</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线 Light</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>在开始之前请首先寻找游戏</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>331</y>
        <width>461</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="main_commitButton">
           <property name="text">
            <string>确 认</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="main_exitButton">
           <property name="text">
            <string>退出程序</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="main_plantplace">
           <property name="text">
            <string>随机放置植物</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>自定义放置植物</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>随机放置僵尸</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>自定义放置僵尸</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>游戏设置</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>801</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>植物相关</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>38</y>
         <width>781</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QCheckBox" name="plantInvincible">
          <property name="text">
           <string>植物无敌</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="nocostsun">
          <property name="text">
           <string>不消耗阳光</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="freeplant">
          <property name="text">
           <string>自由种植</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="nocool">
          <property name="text">
           <string>卡片无冷却</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="nocrater">
          <property name="text">
           <string>不留弹坑</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="noawake">
          <property name="text">
           <string>蘑菇免唤醒</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="nolimitonplant">
          <property name="text">
           <string>紫卡无限制</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="clearplant">
          <property name="text">
           <string>清空场上植物</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>801</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>僵尸相关</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>621</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="zombieInvincible">
          <property name="text">
           <string>僵尸无敌</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="pausesystemzombie">
          <property name="text">
           <string>暂停系统出怪</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="zombiepause">
          <property name="text">
           <string>僵尸不动</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="clearallzombies">
          <property name="text">
           <string>秒杀所有僵尸</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>801</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>其他杂项</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>781</width>
         <height>93</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="3">
         <widget class="QCheckBox" name="lockspade">
          <property name="text">
           <string>鼠标锁定铲子</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="hidespade">
          <property name="text">
           <string>隐藏铲子</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="autocollect">
          <property name="text">
           <string>自动收集</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="unlockallmode">
          <property name="text">
           <string>解锁所有模式</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="passthegame">
          <property name="text">
           <string>直接过关</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QCheckBox" name="pausegame">
          <property name="text">
           <string>暂停游戏</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="hideslot">
          <property name="text">
           <string>隐藏卡槽</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="hidemenu">
          <property name="text">
           <string>隐藏菜单按钮</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="nosun">
          <property name="text">
           <string>不掉阳光</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="unlockglod">
          <property name="text">
           <string>解锁金向日葵</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="allowrunback">
          <property name="text">
           <string>允许后台运行</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLineEdit" name="more">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>781</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>更多功能敬请期待！</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>781</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="suntext"/>
         </item>
         <item>
          <widget class="QPushButton" name="sunchange">
           <property name="text">
            <string>阳光修改</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="quickset">
         <property name="text">
          <string>一键斗蛐蛐</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetall">
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="main_menu">
    <property name="title">
     <string>主界面</string>
    </property>
    <addaction name="setting"/>
   </widget>
   <widget class="QMenu" name="plant_menu">
    <property name="title">
     <string>放置植物</string>
    </property>
    <addaction name="rand_plant_mode_menu"/>
    <addaction name="separator"/>
    <addaction name="custom_plant_mode_menu"/>
   </widget>
   <widget class="QMenu" name="zombie_menu">
    <property name="title">
     <string>放置僵尸</string>
    </property>
    <addaction name="rand_zombie_mode_menu"/>
    <addaction name="separator"/>
    <addaction name="custom_zombie_mode_menu"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="browshelpinfo"/>
   </widget>
   <widget class="QMenu" name="version_menu">
    <property name="title">
     <string>版本信息</string>
    </property>
    <addaction name="browseversion"/>
   </widget>
   <addaction name="main_menu"/>
   <addaction name="plant_menu"/>
   <addaction name="zombie_menu"/>
   <addaction name="help_menu"/>
   <addaction name="version_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="rand_plant_mode_menu">
   <property name="text">
    <string>随机放置模式</string>
   </property>
  </action>
  <action name="custom_plant_mode_menu">
   <property name="text">
    <string>自定义放置模式</string>
   </property>
  </action>
  <action name="rand_zombie_mode_menu">
   <property name="text">
    <string>随机放置模式</string>
   </property>
  </action>
  <action name="custom_zombie_mode_menu">
   <property name="text">
    <string>自定义放置模式</string>
   </property>
  </action>
  <action name="browseversion">
   <property name="text">
    <string>查看版本信息</string>
   </property>
  </action>
  <action name="browshelpinfo">
   <property name="text">
    <string>查看帮助信息</string>
   </property>
  </action>
  <action name="setting">
   <property name="text">
    <string>游戏设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
